rust/pact_matching/src/models/mod.rs

```diff
@@ -9,14 +9,15 @@ use std::default::Default;
 use std::fmt::{Debug, Display, Formatter};
 use std::fs::File;
 use std::hash::{Hash, Hasher};
-use std::io::{self, Error, ErrorKind};
 use std::io::prelude::*;
 use std::io::SeekFrom;
 use std::path::Path;
 use std::str;
 use std::str::from_utf8;
 use std::sync::{Arc, Mutex};
 
+use anyhow::anyhow;
+use anyhow::Context;
```

Comment:
Nitpick: I'd replace with `use anyhow::Context as _;`. It does the same thing, but doesn't bring the `Context` name into scope, while still making the trait's methods usable.

Reply:
Right, makes sense. I'd forgotten about that syntax!